victim/Dockerfile

```diff
@@ -1,25 +1,26 @@
-FROM python:3.8.1-alpine3.11
+FROM python:3.8.2-buster
```

Comment:
I switched out alpine, it's too finicky and we're not going to be concerned about image size anyway since this will all be pre-installed on the VM